public/js/actions/subscriptions.js

```diff
@@ -33,6 +33,40 @@ export function getUserSubscriptions(fetch=api.fetch) {
 }
 
 
+export function getSubsByPayMethod(payMethodUri, fetch=api.fetch) {
```

Comment:
I didn't see a direct test for this action and I think it needs one so that the Ajax logic is covered